test/sql/bgw_launcher.sql

```diff
@@ -284,6 +285,7 @@ SELECT count(pg_terminate_backend(pg_stat_activity.pid)) AS TERMINATED
 FROM pg_stat_activity
 WHERE pg_stat_activity.datname = :'TEST_DBNAME_2'
 AND pg_stat_activity.pid <> pg_backend_pid() \gset
+SELECT pg_sleep(10);
```

Comment:
do we really need to wait 10sec here?

Reply:
No this would be a bad idea. I'm trying to debug this, apparently we don't wait for termination of these backends, might need to add a function for this similar to what we have already to wait for the scheduler start.